src/functionalTest/resources/features/F-209 - Apply NoC Decision/F-209 - Apply NoC Decision.feature

```diff
@@ -0,0 +1,198 @@
+@F-209
+Feature:
+  AS AN invoker of the ApplyNoCDecision API
+  I WANT to check if a NoCRequest has been manually approved
+  SO THAT I can apply the necessary steps to the case in relation to the action being taken
+
+  @S-209.1
+  Scenario: AC2, AC1B Happy path  Apply NoCDecision to remove and replace representation when a NoCRequest has been approved (Happy Path)
+    Given an appropriate test context as detailed in the test data source,
+    Given a user [Richard - with the ability to create a case for a particular jurisdiction within an organisation],
+    And a user [Dil - with a solicitor role for the same jurisdiction within a different organisation from Richard],
+    And a successful call [by Richard to create a case - C1 - on behalf of Mario as the applicant which is auto-assigned to Richard's organisation] as in [F-209_NoC_Case_Creation_By_Richard_With_Assigned_Org_Policies],
+    And a successful call [to remove CREATOR assigned CaseRole] as in [F-209_Delete_Creator_Role],
+    And a successful call [to assign users to the case] as in [F-209_assign_access_to_case],
+    And a wait time of [5] seconds [to allow for Logstash to index the case just created],
+    And the request [by Dil to raise a NoCRequest to become the representative for Mario on C1],
+    When a request is prepared with appropriate values,
+    And the request [intends to apply a NoCDecision to a case],
+    And the request [contains the case record for C1],
+    And the request [contains an approval status of 'Approved']
+    And it is submitted to call the [ApplyNoCDecision] operation of [Manage Case Assignment Microservice],
+    Then a positive response is received,
+    And the response has all the details as expected,
+    And a call [to verify that Richard and anyone else in Richard's organisation has had their access removed] will get the expected response as in [F_209_get_case_users_richard_not_assigned],
+    And a wait time of [5] seconds [to allow for Logstash to re-index the case],
+    And a call [to verify that the supplementary counter has been adjusted to reflect the reduction in the number of users who have access to the case in Richard's organisation] will get the expected response as in [F_209_Search_Case_ES_0_users],
+    And the response [has all fields in the ChangeOrganisationRequest nullified],
+    And the response [to get the case verifies that the PreviousOrganisations field within the Organisation Policy contains a single previous organisation element in which the FromTimeStamp matches the timestamp of When the case was created, And the toTimeStamp is within a couple of seconds of the LastModifiedDate of the case, And the OrganisationName And OrganisationAddress match those of Richard's Organisation].
```

Comment:
If supported in the framework, we could possibly split into multiple `And the response [...]` lines for each of the separate assertions, just make it a bit easier to read